common-ts/src/serializableTypes.ts

```diff
@@ -863,6 +863,112 @@ export class UISerializableOrderActionRecordV2
 		);
 	}
 }
+export class UISerializablePositionHistoryRecord {
```

Comment:
i think its better in the long term if we have a base interface (e.g. `UISerializableOrderActionRecordBase`) that `UISerializableOrderActionRecord` extends from, and it will have everything except `eventType`. `UISerializablePositionHistoryRecord` will then extend from this base interface too

if we were to update the `UISerializableOrderActionRecord` interface in the future, i'm pretty sure we will easily forget to update `UISerializablePositionHistoryRecord` too if we implement it your way...

Reply:
I agree, I originally was trying to do extends but was having trouble. I think the eventType was probably the thing I was missing which you have highlighted so let me adjust for that.